setup.py

```diff
@@ -52,6 +52,7 @@
 ]
 
 install_requires = [
+    'Flask>=1.1.4,<2.0.0',
```

Comment:
with `Flask>=1.1.0`, some tests for min reqs would fail, because of the `SECRET_KEY` apparently being set to `None`

Reply:
Ideally we try to avoid to do this as much as possible, we moved it to `invenio-base` to avoid duplication everywhere.
I am wondering if moving `invenio-base` up, it solves the issue...